README.md

```diff
@@ -0,0 +1,73 @@
+# dnsprove.js 
+
+## Functionalities
+
+- Fetches DNS information of given domain name and type
+- Validates DNS reponse and constructs proofs
+- Submits the proofs into DSNSEC Oracle smart contract
+- Works from both browser and from node.js
+
+## Installing
+
+```
+npm install '@ensdomains/dnsprovejs' --save
+```
+
+## Usage
+
+```js
+var Web3      = require('web3');
```

Comment:
Users should probably get the provider by calling `web3.currentProvider` on the injected web3 object.